docs/pages/blog/date-pickers-stable-v5.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'The Date pickers gets a stable v5 release'
```

Comment:
Let's capitalize it since it's a proper noun (consider "MUI X Date Pickers" vs "a date picker component"), and fix the verb conjugation ("They get"). I would also suggest adding MUI X to the title for branding purposes.

```suggestion
title: 'The MUI X Date Pickers get a stable v5 release'
```

Reply:
We also need to fix the grammar on the card.

Reply:
The naming of the product line is "Date and time pickers" by the way (talking about the work, not the capitalization)

![image](https://user-images.githubusercontent.com/3309670/190431442-4e514a42-8e34-42ef-ab1e-c617fcf441d7.png)

It can be reconsidered but unless we do it I would try to be coherent here

Reply:
Good point, we should use the full name.

```suggestion
title: 'The MUI X Date and Time Pickers get a stable v5 release
```